scanner-darkly/teletype
# Feature/satellite update

#### What does this PR do?

Update satellite to Teletype v4.0.0

#### Provide links to any related discussion on [lines](https://llllllll.co/).

No previous discussion other than a query about the status of alternate firmwares: https://llllllll.co/t/a-users-guide-to-monome-eurorack-firmwares/18031/10  Over on lines I'm 3-foot-1.

#### How should this be manually tested?

Please let me know!  I did the update very mechanically (by forcing a set of merge conflicts), and I'm running a script right now on ansible, controlled by a new-generation grid (confirms a v4 feature) and clocked with an external metro (confirms satellite variant).  But I've never touched Teletype or satellite before, so I'm not really what to do next.  I see that there is a test suite, but for the main `module` firmware, not for ansible.

#### Any background context you want to provide?

At the moment I just want to make you aware that this branch exists and seems to work.  If you think it's worth moving forward from here, I'd appreciate some guidance on testing, and then I'll do the cleanup and doc steps as needed.

#### If the related Github issues aren't referenced in your commits, please link to them here.

#### I have,
